lib/lotus/routing/http_router.rb

```diff
@@ -146,15 +146,16 @@ def no_response(request, env)
       end
 
       private
-      def add_with_request_method(path, method, opts = {}, &app)
-        super.generate(@resolver, opts, &app)
-      end
 
-      def _rescue_url_recognition
+      def _rescue_url_recognition(*args)
```

Comment:
@AlfonsoUceda `*args` aren't used by this method in the end. There isn't any reason to pass them as argument here :wink: